bench/curry.js

```diff
@@ -1,31 +1,56 @@
 const benchmark = require('benchmark');
 const suite = new benchmark.Suite();
 const curry = require('lodash.curry');
+const {util} = require('..');
 
 const lodash = curry(function(a, b){
   return a + b;
 });
 
-const manual = function(a, b){
-  if(arguments.length === 1) return b => manual(a, b);
+function manual(a, b){
+  if(arguments.length === 1) return util.unaryPartial(manual, a);
   return a + b;
-};
+}
+
+function manual3if(a, b, c, d){
+  if(arguments.length === 1) return util.unaryPartial(manual3if, a);
```

Comment:
No. The idea here is to test the performance of `if` vs `switch`. That's why the function is called manual3**if** ^^